app/components/date-time-picker.hbs

```diff
@@ -1,14 +1,25 @@
 <div class="au-o-grid au-o-grid--small">
   <div class="au-o-grid__item au-u-2-6">
-    <AuLabel>Datum</AuLabel>
-    <WuDatepicker class="au-c-datepicker" @onChange={{this.onChangeDate}} @value={{this.date}}/>
+    {{#let (concat this.uniqueId "-date") as |id|}}
```

Comment:
I didn't want to add more unrelated code, but I think adding the [`unique-id` helper](https://emberjs.github.io/rfcs/0659-unique-id-helper.html) would be a better solution. There is a [polyfill](https://github.com/ctjhoa/ember-unique-id-helper-polyfill) for it, but [it's very easy to implement](https://github.com/ctjhoa/ember-unique-id-helper-polyfill/blob/master/addon/helpers/unique-id.js) as well so creating it in app is an option as well.

Reply:
Created a separate PR #90 for this.

Reply:
I've merged #90 :+1: